### PR TITLE
new node publishing & preview

### PR DESCRIPTION
**Pull request by gka**
opening a Draft PR for this

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,12 +4,9 @@
...
         "format": "prettier 'src/**/*.{js,html}' --write",
         "lint": "prettier --check 'src/**/*.{js,html}' && healthier 'src/**/*.{js,html}'",
-        "install": "grunt",
         "watch": "watch 'make assets' assets/",
         "pretest-legacy": "cd test_legacy/build && rollup -c",
-        "test-legacy": "./node_modules/mocha/bin/mocha --require reify",
-        "test": "ava && cd dw.js && npm install && npm test",
-        "test:watch": "ava --watch --verbose"
+        "test-legacy": "./node_modules/mocha/bin/mocha --require reify"
     },
```
Comment (gka) [inline package.json:10]:
```suggestion
        "test": "ava",
        "test:watch": "ava --watch --verbose"
    },
```
I think we want to keep the `ava` tests

File: www/static/js/dw/backend/popupChart.js [hunk 2]
```diff
@@ -1,34 +1,35 @@
-
 define(function() {
-
     return function(chart_id, preview, closeCb) {
-
-        $.getJSON('/api/charts/'+chart_id, function(res) {
-            if (res.status == "ok") {
+        $.getJSON('/api/charts/' + chart_id, function(res) {
```
Comment (gka) [inline www/static/js/dw/backend/popupChart.js:3]: why not use the `v3` api here while we're at it?

File: package.json (hunk 1 above)
Comment (sto3psl) [reply package.json:10]: Good catch. Didn't know that this repo had `ava` tests. Then we should restore the CI test job too.

Comment (sto3psl) [reply package.json:12]: They are back: aa004c1

**Pull request merged**